windows/windows_update_install/prepare_msu_file.yml

```diff
@@ -83,7 +83,7 @@
     fail_msg: >-
       The .msu file of type {{ msu_file_type }} is essential. But it's not found in shared folder: {{ windows_nfs_msu_path }}.
     success_msg: >-
-      The .msu file of type {{ msu_file_type }} is found in shared folder: {{ msu_nfs_path_list[0] }}
+      The .msu file of type {{ msu_file_type }} is found in shared folder.
```

Comment:
I think you can reserve `{{ msu_nfs_path_list }}` but just remove `[0]`, after all `msu_nfs_path_list` is defined.

Reply:
Sure, thanks.